packages/cli/src/lib/system/docker.ts

```diff
@@ -76,6 +87,15 @@ export async function copyArtifactsFromBuildImage(
     }
 
     await runCommand(`docker rm -f root-${imageName}`, quiet);
+
+    if (useBuildkit) {
+      if (removeBuilder) {
+        await runCommand(`docker buildx rm ${imageName}`, quiet);
+      }
+      if (removeImage) {
+        await runCommand(`docker rmi ${imageName}`, quiet);
```

Comment:
This does not seem to require buildx, should it be inside the `buildkit` options in the manifest?

Reply:
Nice catch!